scripts/configure-mingw-w64

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+. "$(dirname "$0")/configure-common"
+
+TOOLCHAIN_FILE="$SOURCE/cmake/Toolchain/Toolchain-mingw-w64-$(uname -m).cmake"
```

Comment:
it will return i686 with an i686 kernel. however of course the output of uname is not very portable. e.g. it will return amd64 on *bsd and kernel architecture doesn't have to match the binary environment.

but I don't care much because nobody will ever call this script on such environments or platforms.

But feel free to change this to something you think is superior later.